components/brave_sync/brave_profile_sync_service_impl.h

```diff
@@ -10,6 +10,8 @@
 #include <string>
 #include <vector>
 
+#include "base/one_shot_event.h"
```

Comment:
seems like it is not needed anymore

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/4588/commits/faae417c8590f9732feec99c5b4df889845ab060